neo/rawio/neuralynxrawio/neuralynxrawio.py

```diff
@@ -183,33 +222,24 @@ def _parse_header(self):
         event_annotations = []
 
         if self.rawmode == "one-dir":
-            filenames = sorted(os.listdir(self.dirname))
-            dirname = self.dirname
+            filenames = sorted([os.path.join(self.dirname, f) for f in os.listdir(self.dirname)
```

Comment:
This has become very complicated. I would actually keep the original logic and just use `listdir` here.